libs/eo/faq/shell/src/lib/eo-faq-shell.component.ts

```diff
@@ -0,0 +1,114 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ChangeDetectionStrategy, Component, NgModule } from '@angular/core';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  selector: 'eo-faq-shell',
+  styles: [
+    `
+      :host {
+        display: block;
+        width: calc(200 * var(--watt-space-xs));
+      }
+    `,
+  ],
+  template: `
+    <div class="watt-space-stack-xl">
+      <div class="watt-space-stack-m">
+        <a (click)="scrollToTargetElement(what_is_energy_origin)"
+          >What is Energy Origin?</a
+        >
+      </div>
+      <div class="watt-space-stack-m">
+        <a (click)="scrollToTargetElement(who_can_access_the_platform)"
+          >Who can access the platform?</a
+        >
+      </div>
+      <div class="watt-space-stack-m">
+        <a (click)="scrollToTargetElement(where_does_the_data_come_from)"
+          >Where does the data come from?</a
+        >
+      </div>
+      <div class="watt-space-stack-m">
+        <a (click)="scrollToTargetElement(how_can_i_influence_the_development)"
+          >How can I influence the development?</a
+        >
+      </div>
+      <div class="watt-space-stack-m">
+        <a (click)="scrollToTargetElement(where_can_i_read_more)"
+          >Where can I read more?</a
+        >
+      </div>
+    </div>
+
+    <h3 #what_is_energy_origin>What is Energy Origin?</h3>
+    <p>
+      The aim of the Energy Origin platform is to create transparency on the
+      origin of energy for all energy consumers and to facilitate a market for
+      for green energy. Currently, we are working on data related to electricity
+      consumption. Upon login, you will soon be able to access your hourly
+      electricity declaration and corresponding emissions for 2021.
+    </p>
+
+    <h3 #who_can_access_the_platform>Who can access the platform?</h3>
+    <p>
+      Currently, the platform only offers company login via NemID. Later on, it
+      will be possible for private individuals to login via NemID / MitID.
+    </p>
+
+    <h3 #where_does_the_data_come_from>Where does the data come from?</h3>
+    <p>
+      All data related to metering points, consumption and production is served
+      by the Energinet DataHub. Data related to the origin of energy and
+      corresponding emissions stems from Energi Data Service.
+    </p>
+
+    <h3 #how_can_i_influence_the_development>
+      How can I influence the development?
+    </h3>
+    <p>
+      You are more than welcome to participate in our LinkedIn user group, a
+      digital forum for users, in which we post sketches, questions and gather
+      ideas and suggestions for new development and improvements. It’s
+      non-binding and you can either just follow along without getting involved
+      or comment when you have time.
+    </p>
+
+    <h3 #where_can_i_read_more>Where can I read more?</h3>
```

Comment:
suggestion: Did you consider using a directive to dynamically apply a fragment (element ID) based on the headline text and to add smooth scroll support?